submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/migrations/DatabaseChangeLog.kt

```diff
@@ -34,20 +35,32 @@ class DatabaseChangeLog {
             ensureIndex(Index().on(SUB_SUBMITTER, ASC))
             ensureIndex(Index().on("$SUB_SECTION.$SEC_TYPE", ASC))
             ensureIndex(Index().on(SUB_RELEASE_TIME, ASC))
-            ensureIndex(TextIndex().onField(SUB_TITLE).build())
+            ensureIndex(submissionTitleIndex())
             ensureIndex(Index().on(SUB_RELEASED, ASC))
         }
 
         template.indexOps(SubmissionRequest::class.java).apply {
             ensureIndex(Index().on(SUB_ACC_NO, ASC))
             ensureIndex(Index().on(SUB_ACC_NO, ASC).on(SUB_VERSION, ASC))
-            ensureIndex(Index().on("submission.$SUB_SECTION.$SEC_TYPE", ASC))
-            ensureIndex(Index().on("submission.$SUB_ACC_NO", ASC))
-            ensureIndex(Index().on("submission.$SUB_OWNER", ASC))
-            ensureIndex(Index().on("submission.$SUB_SUBMITTER", ASC))
-            ensureIndex(Index().on("submission.$SUB_RELEASE_TIME", ASC))
-            ensureIndex(TextIndex().onField("submission.$SUB_TITLE").build())
-            ensureIndex(Index().on("submission.$SUB_RELEASED", ASC))
+            ensureIndex(Index().on("$SUB.$SUB_SECTION.$SEC_TYPE", ASC))
+            ensureIndex(Index().on("$SUB.$SUB_ACC_NO", ASC))
+            ensureIndex(Index().on("$SUB.$SUB_OWNER", ASC))
+            ensureIndex(Index().on("$SUB.$SUB_SUBMITTER", ASC))
+            ensureIndex(Index().on("$SUB.$SUB_RELEASE_TIME", ASC))
+            ensureIndex(submissionRequestTitleIndex())
+            ensureIndex(Index().on("$SUB.$SUB_RELEASED", ASC))
         }
     }
+
+    private fun submissionTitleIndex() =
```

Comment:
1. why we are adding a textIndex on both submission title and attributes values?
2. We need to add new migration not update existing one. as they will not be executed